db/disk.go

```diff
@@ -44,3 +49,72 @@ func initDisks(tx *sql.Tx) error {
 
 	return err
 }
+
+func (d DB) CreateDisk(vendor string,model string,
+	machine *Machine, capacity int, ssd bool) error {
+	_, err := d.sql.Exec(`
+		INSERT INTO Disks(vendor, model, machine, capacity, ssd)
+		VALUES(
+			?,
+			?,
+			?,
+			?,
+			?
+			)`,
+		 vendor, model, machine.Id, capacity, ssd)
+		 //fmt.Printf("", machine)
+
+	return err
+}
+
+
+func (d DB) GetDisks() ([]Disk, error){
+	rows, err := d.sql.Query(`
+		SELECT id, vendor, model, machine, capacity, ssd
+		FROM Disks 
+		ORDER BY capacity DESC
+
+	`)
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+
+	disks := []Disk{}
+	for rows.Next() {
+		k := Disk{db: &d}
+		var machine int
+			err = rows.Scan(&k.id, &k.Vendor,
```

Comment:
It would be useful to add a blank line between `var machine int` and `err = rows.Scan`.